src/error_define.go

```diff
@@ -131,6 +132,7 @@ func (e errorIndex) String(args ...string) string {
 		/*ErrUserStatus*/ "Please check this user account in this device",
 		/*ErrUserAdmin*/ "The user privilege is not administrator",
 		/*ErrUserPrivilege*/ "The user privilege could not configure this action to the device",
+		/*ErrWrongPrivilege*/ "Wrong privilege",
```

Comment:
Consider expanding error message - something like "Wrong privilege for this action"

Reply:
Error message changed.